src/Query.ts

```diff
@@ -204,8 +204,8 @@ export default class Query {
         });
       },
       bindings: () => {
```

Comment:
Why not have a 'boolean' function, next to the 'bindings' function? That'd make the interface more consistent and the implementation simpler